src/music.cpp

```diff
@@ -0,0 +1,106 @@
+#include "music.h"
+#include "debug.h"
+
+namespace music
+{
+
+std::vector<std::pair<music_id, bool>> music_id_list { std::make_pair( mp3, false ), std::make_pair( instrument, false ), std::make_pair( sound, false ), std::make_pair( title, true ) };
+
+bool is_active_music_id( music_id data )
+{
+    for( std::pair<music_id, bool> &pair : music_id_list ) {
```

Comment:
This won't change with a map.
```suggestion
    for( const std::pair<music_id, bool> &pair : music_id_list ) {
```

Reply:
Yeah, I changed that from std::pair<music_id, bool> to std::pair<const music_id, bool>